tests/themes/test_schema.py

```
@@ -0,0 +1,63 @@
+
+def _base():
+    return {"tokens": {}, "structure": {}, "effects": [], "requires": ["assistant", "launcher"]}
+
+def test_valid_minimal_config_passes():
+    cfg = _base()
+    cfg["tokens"] = {"--color-shell-bg": "#000000", "--color-accent": "rgb(0,255,70)"}
+    out = validate_theme_config(cfg)
+    assert out["tokens"]["--color-shell-bg"] == "#000000"
+
+def test_unknown_token_key_rejected():
+    cfg = _base(); cfg["tokens"] = {"--evil-key": "#000"}
```

Ruff (0.15.14)

[error] Multiple statements on one line (semicolon)

(E702)